backend/utils/authUtils.js

```diff
@@ -1,15 +1,25 @@
 const AWS = require('aws-sdk')
-var express = require("express")
-const AmazonCognitoIdentity = require('amazon-cognito-identity-js');
+const Axios = require('axios')
+const jwkToPem = require('jwk-to-pem')
+const crypto = require('crypto')
+const AmazonCognitoIdentity = require('amazon-cognito-identity-js')
+
 const AWS_ACCESS_KEY_ID = "AKIATNLYKFHXVBMV5DOO"
 const AWS_SECRET_ACCESS_KEY = "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"
 const USERPOOLID = "us-west-2_LJltK6RSZ"
 const CLIENTID = "7docu6553pf9vi5ipv1ig1huns"
+const COGNITO_ISSUER = `https://cognito-idp.us-west-2.amazonaws.com/${USERPOOLID}`
 const userPool = new AmazonCognitoIdentity.CognitoUserPool({
     UserPoolId: USERPOOLID,
     ClientId: CLIENTID
 });
 
+//AWS cognito generates two pairs of RSA keys (a public and local key) for each userpool,
+// the public key available at the following URL, and the local key is generated when user logs in
+const url = COGNITO_ISSUER + '/.well-known/jwks.json'
```

Comment:
Codacy has a fix for the issue: [Replace `'/.well-known/jwks.json'` with `"/.well-known/jwks.json";`](https://app.codacy.com/gh/cpen321project/cpen321project/pullRequest?prid=9852555)
```suggestion
const url = COGNITO_ISSUER + "/.well-known/jwks.json";
```